akka-docs/src/main/paradox/stream/operators/Source-or-Flow/reduce.md

```diff
@@ -15,6 +15,17 @@ Start with first element and then apply the current and next value to the given
 Start with first element and then apply the current and next value to the given function, when upstream
 complete the current value is emitted downstream. Similar to `fold`.
 
+## Example
+
+`reduce` will take a function and apply it on the incoming elements in the Stream and only emits its result when upstream completes.
+Here, it will add the incoming elements.
+Scala
```

Comment:
There needs to be a newline here, otherwise the tabs won't work:

```suggestion

Scala
```

(you can generate the docs with `sbt paradox` to check them)